expression/distsql_builtin.go

```diff
@@ -917,9 +917,9 @@ func getSignatureByPB(ctx sessionctx.Context, sigCode tipb.ScalarFuncSig, tp *ti
 	case tipb.ScalarFuncSig_CharLengthUTF8:
 		f = &builtinCharLengthUTF8Sig{base}
 	case tipb.ScalarFuncSig_Concat:
-		f = &builtinConcatSig{base, 65536}
+		f = &builtinConcatSig{base, 67108864}
```

Comment:
Could this be a Constant, along with https://github.com/pingcap/tidb/blob/945f7cc4a04dd09367674b4e5d76f1602812da15/sessionctx/variable/sysvar.go#L580?